content/docs/setup/kubernetes/upgrade/notice/index.md

```diff
@@ -33,4 +33,7 @@ For an overview of new features introduced with Istio 1.1, please refer to the [
 
 ## Security
 
-- RBAC configuration has been modified to implement cluster scoping.  The `RbacConfig` resource has been replaced with the `ClusterRbacConfig` resource. Refer to [Migrating `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrade/steps/#migrating-from-rbacconfig-to-clusterrbacconfig) for migration instructions.
+- The `RbacConfig` resource has been replaced with the `ClusterRbacConfig` resource in 1.1. You can ignore this if you have already
```

Comment:
In Istio 1.1, the `RbacConfig` resource was replaced with the `ClusterRbacConfig` resource.